app/controllers/index.js

```diff
@@ -1,14 +1,18 @@
-import { action } from '@ember/object';
 import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 import { all } from 'rsvp';
 
 export default class IndexController extends Controller {
-  contributorsList = '';
+  @tracked mergedPRs = [];
+  @tracked newPRs = [];
+  @tracked mergedRFCs = [];
+  @tracked newRFCs = [];
+  @tracked contributorsList = '';
 
   @action
   async getContributors() {
-    const { mergedPulls } = this.model;
-    const fetchRequests = mergedPulls.map(pull => pull.get('user'));
+    const fetchRequests = this.mergedPRs.map(pullRequest => pullRequest.user);
```

Comment:
If all we care about is the user's handle, we no longer need to make a fetch request. We can directly access the information via `pullRequest.user.login`.

But if we do want to get the user's name, we will want to continue to make the request.